### ISSUE TITLE
Update va-file-input component to V3

### ISSUE THREAD

**Issue by pmclaren19** [department-of-veterans-affairs/va.gov-team#79156]
## Background Context
Currently there are a few issues when updating the the VAFileInput component from V1 to V3.

**Issues:**
[PLATFORM DESIGN BUG] Unable to upload multiple files at the same time https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2549 has been split into 3 tickets...
- va-icon-button - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2641 (needed for file input)
- update file input - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2642 (needs to happen before multiple file input)
- multiple file input - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2643

[PLATFORM DESIGN BUG] - Unable to delete a file once its been added https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2511 this work has been added into this ticket...
- [USWDS - Feature: File Input on Cancel resets input](https://github.com/uswds/uswds/issues/5212#top) 

[PLATFORM DESIGN BUG] - There isnt a card design that matches and works well with this component https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1578 this work has been added to this ticket...
- [Staging Review: (DS) Converted File Input Component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77552)

### Bug
- The above 3 tickets need to be resolved before we can use the V3 version of this component

## Design
- Add here

## Engineering 
- Add here

## Out of scope
- Add here

## Open questions
- Add here

## Tasks
- [ ] Update cypress tests to no longer be skipped in src/applications/claims-status/tests/e2e/02.claim-files.v2.cypress.spec.js

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] The VaFileInput component on AddFilesForm is updated to V3
- [ ] The VaFileInput component on AddFilesFormOld is updated to V3
- [ ] Document Request Page uses the V3 version of the VaFileInput
- [ ] The Files tab uses the V3 version of the VaFileInput

**Comment by pmclaren19** [comment]
Created a new ticket for this work - https://github.com/department-of-veterans-affairs/va.gov-team/issues/87835 this ticket can be closed now!
